internal/node/test/npm_package_bin.spec.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs');
 const path = require('path');
-const runfiles = require(process.env['BAZEL_NODE_RUNFILES_HELPER']);
+const runfiles = require('@bazel/runfiles');
```

Comment:
hmm, it's a bit confusing that there is no such package on npm. But I think this is probably the best name to use (could be rules_nodejs_runfiles maybe, as this is part of the builtin?)

but I do like the idea of linking it at `node_modules/@bazel/runfiles` rather than use an environment variable.